.github/workflows/test-library.yml

```diff
@@ -679,15 +689,25 @@ jobs:
         docker buildx use proxypybuilder
         docker buildx inspect
         docker buildx ls
-    - name: Set PROXYPY_CONTAINER_VERSION
-      run: |
-        echo "PROXYPY_CONTAINER_VERSION=$(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .)" > $GITHUB_ENV
-    - name: Build container
-      run: |
-        make container-buildx \
-          -e PROXYPY_PKG_PATH='dist/${{ needs.pre-setup.outputs.wheel-artifact-name }}' \
-          -e BUILDX_TARGET_PLATFORM='${{ matrix.targetplatform }}' \
-          -e PROXYPY_CONTAINER_VERSION='${{ env.PROXYPY_CONTAINER_VERSION }}'
+    - name: Build container & Push to GHCR
+      run: >-
+        docker images;
+        make container-buildx
+        -e PROXYPY_PKG_PATH='dist/${{
+          needs.pre-setup.outputs.wheel-artifact-name
+        }}'
+        -e BUILDX_TARGET_PLATFORM='${{ matrix.targetplatform }}'
+        -e PROXYPY_CONTAINER_VERSION='${{
```

Comment:
In case you want to have a shared image on GHCR, here's an example of having a unique workflow run based tag to be reused across jobs: https://github.com/ansible/creator-ee/blob/8f2ef3e/.github/workflows/release.yml#L114-L118.

Reply:
Thank you, can borrow code example from here.  Here we'll need one image / arch / workflow.  Then push all arch images (with common tag) to registries.

Reply:
Idea is to get this result (similar to `python-alpine`)

<img width="1253" alt="Screen Shot 2021-11-29 at 1 37 56 AM" src="https://user-images.githubusercontent.com/126065/143784144-5e7d9261-5ec8-4a06-b401-7d2a4135af4b.png">

Currently we are here

<img width="1253" alt="Screen Shot 2021-11-29 at 1 44 31 AM" src="https://user-images.githubusercontent.com/126065/143784353-8644d5a3-bcfe-444a-90da-0ee122a3c395.png">

Reply:
This PR now pushes to GHCR.  Containers available here https://github.com/abhinavsingh/proxy.py/pkgs/container/proxy.py

But looks like GHCR doesn't support list manifests and we can't get something like DockerHub on GHCR.  Will dig more later as part of a separate PR.  Have to get back eventually to push to DockerHub which I have skipped in this PR.

Reply:
FWIW I only use GHCR as a staging area before publishing to more "official" registries, I don't think it's meant to be a full replacement of those.